plugins/deploy_waitlist/lib/samson_deploy_waitlist/samson_plugin.rb

```diff
@@ -0,0 +1,141 @@
+module SamsonDeployWaitlist
+  class Engine < Rails::Engine
+  end
+end
+
+# To add in your own UI for a hook, e.g., for the stage edit page:
+# - Create app/views/samson_deploy_waitlist/_my_fields.html.erb
+# - Add this line to this file, note the lack of leading underscore and extension:
+#   Samson::Hooks.view :stage_form, 'samson_deploy_waitlist/my_fields'
+
+
+#Samson::Hooks.view :stage_form, '<view path>'
+
+Samson::Hooks.view :stage_show, 'samson_deploy_waitlist/deployers'
```

Comment:
lets remove the stuff we're not actively using. They'll be potentially out of date with newer versions